elements/rh-tabs/rh-tab.css

```diff
@@ -0,0 +1,154 @@
+#rhds-container {
+  display: contents;
+
+  --_button-text-color: var(--rh-tabs-link-color, var(--rh-color-text-secondary-on-light, #6a6e73));
+}
+
+:host([active]) #rhds-container {
+  --_button-text-color: var(--rh-tabs-link-color, var(--rh-color-text-primary-on-light, #151515));
+  --_button-after-border-color: var(--rh-tabs-link-after-border-color, var(--rh-color-accent-brand-on-light, #ee0000));
+}
+
+:host([active][theme="base"]) #rhds-container {
+  --_button-after-border-color: var(--rh-tabs-border-color, var(--rh-color-border-interactive-on-light, #0066cc));
+}
+
+.dark {
+  --_button-text-color: var(--rh-tabs-link-color, var(--rh-color-text-secondary-on-dark, #d2d2d2));
+}
+
+:host([active]) .dark {
+  --_button-text-color: var(--rh-tabs-link-color, var(--rh-color-text-primary-on-dark, #ffffff));
+  --_button-after-border-color: var(--rh-tabs-link-after-border-color, var(--rh-color-accent-brand-on-dark, #ff3333));
+}
+
+:host([active][theme="base"]) .dark {
+  --_button-after-border-color: var(--rh-tabs-border-color, var(--rh-color-border-interactive-on-dark, #73bcf7));
+}
+
+:host([box]) #rhds-container {
+  --_inactive-tab-background-color: var(--rh-color-surface-light, #f0f0f0);
+}
+
+:host([box]) .dark {
+  --_inactive-tab-background-color: var(--rh-color-surface-dark, #3c3f42);
+}
+
+button {
+  color: var(--_button-text-color);
+  padding-block-start: var(--rh-tabs-link-padding-block-start, var(--rh-spacer-lg, 16px));
+  padding-inline-end: var(--rh-tabs-link-padding-inline-end, var(--rh-spacer-lg, 16px));
+  padding-block-end: var(--rh-tabs-link-padding-block-end, var(--rh-spacer-lg, 16px));
+  padding-inline-start: var(--rh-tabs-link-padding-inline-start, var(--rh-spacer-lg, 16px));
```

Comment:
Default and box tabs should have 32px for inline padding. In the default variant only, the first tab should have `padding-inline-start: 0`, and the last tab should have `padding-inline-end: 0`.